packages/ndla-ui/src/Article/Article.jsx

```diff
@@ -120,12 +120,10 @@ export const Article = ({
     .abbreviation;
 
   let authors = creators;
-  if (Array.isArray(authors) && authors.length === 0) {
-    authors = rightsholders;
-  }
   if (Array.isArray(authors) && authors.length === 0) {
     authors = processors;
   }
+  const suppliers = rightsholders.length ? rightsholders : undefined;
```

Comment:
Vi bruker Article direkte i ndla-frontend så her måtte eg legge til bruk av suppliers for at det skulle vises.